youtube_dl/postprocessor/common.py

```diff
@@ -62,7 +64,10 @@ def try_utime(self, path, atime, mtime, errnote='Cannot update utime of file'):
             self._downloader.report_warning(errnote)
 
     def _configuration_args(self, default=[]):
-        return cli_configuration_args(self._downloader.params, 'postprocessor_args', default)
+        args = self._downloader.params.get('postprocessor_args', {})
+        if isinstance(args, list):  # for backward compatibility
+            args = {'default': args}
+        return cli_configuration_args(args, self.PP_NAME.lower(), args.get('default', []))
```

Comment:
`cli_configuration_args` is supposed to be used with `params` dict, if you are working with `postprocessor_args` on your own you should not use it.

Reply:
what should I use here then?

Reply:
Something like this (not tested):
```python
args = self._downloader.params.get('postprocessor_args')
if args is None:
    return default
if isinstance(args, (list, tuple)):  # for backward compatibility
    return args
assert isinstance(args, dict)
pp_args = args.get(pp_key)
if pp_args is not None:
    return pp_args
pp_args = args.get('default')  # for backward compatibility
if pp_args is not None:
    return pp_args
return default
```

Reply:
That is pretty much a copy-paste of `cli_configuration_args`. Anyway, I have changed it

Reply:
Actually, it's not. You still ignore `default` parameter of `_configuration_args`.